README.md

```diff
@@ -1,2 +1,46 @@
 # voucher-service
```

Comment:
Could you change this 'voucher-service' to 'A Voucher Management as a Service Application'